priv/js/sortable.js

```diff
@@ -180,7 +188,12 @@ function () {
   }, {
     key: "getValues",
     value: function getValues() {
-      return Array.from(this.items.map(function(item) { return item.children[1].firstChild.innerHTML; }));
+      return Array.from(this.items.map(function(item) {
+        let text = item.children[1].firstChild.innerHTML;
+        let bind = item.getAttribute('data-bind');
+        if (bind) return { 'text': text, 'bind': bind };
```

Comment:
Nitrogen internal JSON Bind Protocol, where `text` is a key and `bind` is a base64 encoded BERT term.